cabot/metricsapp/api/base.py

```diff
@@ -7,28 +7,64 @@
 logger = get_task_logger(__name__)
 
 
-def _get_error_message(check, metric_name, value, check_value):
-    return u'{}: {:0.1f} {} {:0.1f}'.format(metric_name, value, check.check_type, check_value)
+def _get_warning_message(check, series_name, value):
+    """
+    Construct a warning message:
+
+        Format:   WARNING <series_name>: <value> not <comparator> <threshold>
+        Example:  WARNING foo.service.errors: 100 not < 50
+
+    """
+    fmt = u'WARNING {}: {:0.1f} not {} {:0.1f}'
+    return fmt.format(series_name, value, check.check_type, check.warning_value)
+
+
+def _get_high_alert_message(check, series_name, value):
+    """
+    Construct a high-alert error message. The message will differ based on whether
+    the check requires N > 1 consecutive failed points.
+
+    For a single failed point:
+
+        Format:   <importance> <series_name>: <value> not <comparator> <threshold>
+        Example:  ERROR foo.service.errors: 100 not < 50
+
+    For N consecutive failed points:
 
+        Format:   <importance> <series_name>: <value> not <comparator> <threshold>
+        Example:  CRITICAL foo.service.errors: 10 adjacent points not < 50
 
-def _point_failure_check(check_type, check_value, value):
+    """
+    if check.consecutive_failures == 1:
+        fmt = u'{} {}: {:0.1f} not {} {:0.1f}'
+        return fmt.format(check.high_alert_importance, series_name, value,
+                          check.check_type, check.high_alert_value)
+    else:
+        fmt = u'{} {}: {} adjacent points not {} {:0.1f}'
```

Comment:
nit: i think "consecutive" is a little better wording than "adjacent"?

Reply:
Sure, I can change.  Was trying to save precious space in the check history table.  No big deal tho.